apps/researcher/src/messages/en/messages.json

```diff
@@ -227,6 +228,13 @@
     "possibly": "Possibly",
     "probably": "Probably"
   },
+  "MotivationInput": {
+    "openButton": "Add motivation",
+    "editButton": "Edit motivation",
+    "description": "In this motivation field you can add an extra motivation about and only about the above field. It will only be display at the provenance section and will affect the search results.",
```

Comment:
1. `displayed`?
2. What does 'and will affect the search results.' mean?

Reply:
Good question, I copied this from the design. I will ask Bas

Reply:
For now I have completely rewritten this description. Without saying anything about what we do with this information. This can be updated by Camiel if needed.